### PR TITLE
Reword

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: src/library/scala/math/Ordering.scala [hunk 1]
```diff
@@ -370,17 +370,17 @@ object Ordering extends LowPriorityOrderingImplicits {
...
     * consistent with `java.lang.Float.compare`.
     *
-    * Scala 2.10.0 changed the implementation of `lt`, `equiv`, `min` etc to be
-    * IEEE 754 compliant, while keeping `compare` method NOT compliant,
-    * creating an internally inconsistent instance. IEEE 754 specified that
-    * `0.0F == -0.0F`. In addition, all comparisons with `Float.NaN` must return
+    * Scala 2.10.0 changed the implementation of `lt`, `equiv`, `min`, etc., to be
+    * IEEE 754 compliant, while keeping the `compare` method NOT compliant,
+    * creating an internally inconsistent instance. IEEE 754 specifies that
+    * `0.0F == -0.0F`. In addition, it requires all comparisons with `Float.NaN` return
```
Comment (eed3si9n) [inline src/library/scala/math/Ordering.scala:381]: 👍

File: src/library/scala/math/Ordering.scala [hunk 2]
```diff
@@ -370,17 +370,17 @@ object Ordering extends LowPriorityOrderingImplicits {
...
     * `false` thus `0.0F < Float.NaN`, `0.0F > Float.NaN`, and
-    * `Float.NaN == Float.NaN` all yield `false`, analogous to `None`.
+    * `Float.NaN == Float.NaN` all yield `false`, analogous calling `exists` on `None`.
```
Comment (eed3si9n) [inline src/library/scala/math/Ordering.scala]: I am saying that `NaN` represents an error state that propagates like `None` does in `flatMap`. Not sure bringing up `exists` makes that clearer.

File: src/library/scala/math/Ordering.scala [hunk 3]
```diff
@@ -454,26 +454,26 @@ object Ordering extends LowPriorityOrderingImplicits {
...
-    "  The new ordering does not affect the sorting, placing NaN at the end.\n" +
-    "  However, methods such as `lt`, `min`, and `equiv` now match `compare`\n" +
-    "  instead of having IEEE 754 behavior for -0.0F and NaN.\n" +
-    "  Import Ordering.Float.IeeeOrdering to retain the previous behavior.\n" +
-    "  See also https://www.scala-lang.org/api/current/scala/math/Ordering$$Float$.html", "2.13.0")
+    "  The default implicit ordering for floats now maintains consistency\n" +
+    "  between its `compare` method and its `lt`, `min`, `equiv`, etc., methods,\n" +
+    "  at the cost of nonconforming to IEEE 754's behavior for -0.0F and NaN.\n" +
+    "  Import Ordering.Float.IeeeOrdering to recover the previous behavior.\n" +
+    "  See also https://www.scala-lang.org/api/current/scala/math/Ordering$$Float$.html.", "2.13.0")
```
Comment (eed3si9n) [inline src/library/scala/math/Ordering.scala]: I think the first thing we should clarify is that sort ordering is actually unaffected between 2.12 and 2.13.
Also "at the cost of" sounds like a value judgement implying IeeeOrdering is _better_, but it's difficult to say if it is in all situations.

File: src/library/scala/math/Ordering.scala (hunk 2 above)
Comment (dwijnand) [reply src/library/scala/math/Ordering.scala]: The mention of `flatMap` works too.  I just thought it was left a little too unclear to just say "analogous to `None`".

File: src/library/scala/math/Ordering.scala (hunk 3 above)
Comment (dwijnand) [reply src/library/scala/math/Ordering.scala]: It didn't mean it as any judgement of "better".  Just that it's a cost/reward thing, not a 1 size fits all.

And I now understand what the "The new ordering does not affect the sorting, placing NaN at the end" bit means now, with your comments.  I dropped it as... it seemed unaffected, and so irrelevant for the migration notes.

**Review by eed3si9n — APPROVED** [review]
Thanks

**Pull request merged**